### PR TITLE
✨ LLM : LIVE-4018 : Braze content cards

### PR DESCRIPTION
**Pull request by cgrellard-ledger**
* 🛠  First step to setup Content card in LLM

* lint

* 🛠  Update files Architecture + Add dynamic Content in the app's Store

* 🚚 Move files

* revome unsed attribute

* 🛠 Rework Braze with hook

### 📝 Description

Content cards [LIVE-4018]

### ❓ Context

- **Impacted projects**: `ledger-live-mobile` `ui:native` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [x] **Atomic delivery** 
- [x] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

[LIVE-4018]: https://ledgerhq.atlassian.net/browse/LIVE-4018?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/ui/packages/native/storybook/stories/Cards/Cards.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,99 @@
+/* eslint-disable no-console */
```
Comment (nparigi-ledger) [inline libs/ui/packages/native/storybook/stories/Cards/Cards.stories.tsx:1]: not needed anymore

File: apps/ledger-live-mobile/src/components/RootNavigator/WalletTabNavigator.tsx [hunk 2]
```diff
@@ -64,6 +64,7 @@ export default function WalletTabNavigator() {
           sceneContainerStyle={{ backgroundColor: "transparent" }}
           screenOptions={{
             lazy: walletNftGalleryFeature?.params?.lazyLoadScreens ?? true,
+            swipeEnabled: false, // For Contents Cards issue
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/components/RootNavigator/WalletTabNavigator.tsx:67]: :cry:

File: apps/ledger-live-mobile/src/dynamicContent/dynamicContent.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    location: LocationContentCard.Asset,
+    image: card.extras.image,
+    link: card.extras.link,
+    cta: card.extras.cta,
+    assets: card.extras.assets ?? "",
+    displayOnEveryAssets: Boolean(card.extras.displayOnEveryAssets) ?? false,
+  } as AssetContentCard);
+
+const useDynamicContent = () => {
+  // const dynamicContentFeature = useFeature("dynamicContent");
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/dynamicContent/dynamicContent.tsx:49]: forgot comment

File: apps/ledger-live-mobile/src/screens/Settings/General/CarouselRow.tsx [hunk 4]
```diff
@@ -1,27 +1,33 @@
...
+          checked ? [] : [...walletCards, ...assetsCards].map(card => card.id),
         ),
-      ),
-    [dispatch],
+      );
+      track("toggle_clicked", {
+        toggle: "Content Cards",
+        enabled: checked,
+        screen: ScreenName.GeneralSettings,
+      }); // TODO
```
Comment (nparigi-ledger) [inline apps/ledger-live-mobile/src/screens/Settings/General/CarouselRow.tsx:28]: Todo is not really clear :grey_question:

File: libs/ui/packages/native/src/components/Cards/CardA/ColoredGradient.tsx [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+import styled from "styled-components/native";
+
+const StyledSVG = styled(Svg)`
+  position: absolute;
+  bottom: 0;
+  width: 100%;
+  height: 100%;
+`;
+
+export type Props = React.ComponentProps<typeof Svg> & {
```
Comment (nparigi-ledger) [inline libs/ui/packages/native/src/components/Cards/CardA/ColoredGradient.tsx:12]: Accept any props of Svg, but they are not used after apart from "style"

**Comment by nparigi-ledger** [comment]
What is the difference between CardA and CardB ?

**Comment by mcayuelas-ledger** [comment]
CardA vs CardB @nparigi-ledger  😊
Screenshot 2022-11-29 at 11 09 09
Screenshot 2022-11-29 at 11 09 30


We are aware that the names are too generic but it is difficult to name them

**Review by mlegall-ledger — APPROVED** [review]

**Pull request merged**